imsv-docs-astro/src/content/docs/guides/authentication.mdoc

```diff
@@ -129,7 +129,22 @@ authenticated by supplying the access token in the `Authorization` header.
     {% /code %}
 
 3.  [Submit the signed challenge along with the
-    signature](/api-reference/login) to get the access token.
+    signature](/api-reference/login) to get the access token and a refresh token.
+    The refresh token is used to obtain a new access token when the current
+    access token expires, ensuring continuous access without requiring the user to log in again.
 4.  The access token should be used for subsequent requests for protected
     resources from the Immersve API by supplying it in the `Authorization`
     header.
+
+## Use Refresh Tokens
+
+The refresh token is a long-lived token that can be used to obtain a new access token without requiring the user to re-authenticate.
+
+Managing Refresh Token
+- Before making a request, check the expiry of the access token. If the token is expired, use the refresh token to obtain a new access token.
+- To obtain a new access token, call the [Auth Token Exchange API](/api-reference/auth-token-exchange) using the refresh token you received during authentication. This will return a new access token and refresh token to be exchanged when the new access token expires
```

Comment:
```suggestion
- To obtain a new access token, call the [Auth Token Exchange API](/api-reference/auth-token-exchange) using the refresh token you received during authentication. This will return a new access token and refresh token to be exchanged when the new access token expires.
```

Reply:
Done 😃